client/asset/btc/btc.go

```diff
@@ -1143,20 +1134,17 @@ func (btc *ExchangeWallet) FundingCoins(ids []dex.Bytes) (asset.Coins, error) {
 // Unlock unlocks the ExchangeWallet. The pw supplied should be the same as the
 // password for the underlying bitcoind wallet which will also be unlocked.
 func (btc *ExchangeWallet) Unlock(pw string) error {
-	return btc.wallet.Unlock(pw)
+	return btc.node.Unlock(pw)
 }
 
 // Lock locks the ExchangeWallet and the underlying bitcoind wallet.
 func (btc *ExchangeWallet) Lock() error {
-	if btc.node.Disconnected() {
-		return asset.ErrConnectionDown
-	}
-	return btc.wallet.Lock()
+	return btc.node.Lock()
```

Comment:
Now that this is `node` this reads awkwardly. What are we locking? I would argue that prefixing some of these methods with `Wallet` would be good. i.e. `btc.node.WalletLock()` There may also be an `AccountLock` at some point in time, so prepares for that as well. Just my opinion though.